lib/workers/repository/finalise/repository-statistics.ts

```diff
@@ -33,3 +40,50 @@ export function runRenovateRepoStats(
   }
   logger.debug({ stats: prStats }, `Renovate repository PR statistics`);
 }
+
+export function branchCacheToMetadata({
```

Comment:
```suggestion
function branchCacheToMetadata({
```
don't export if no public usage